test/test_all.py

```diff
@@ -1,26 +1,31 @@
 #!/usr/bin/env python3
+from test_contributors import TestContributors
+from test_cache import TestLRU
+from test_tileset import TilesetTest
+from test_settings import SettingsTest
+from test_rendertileset import RendertileSetTest
+from test_tileobj import TileTest
 import unittest
 
 # For convenience
-import sys,os,logging
+import sys
+import os
+import logging
 sys.path.insert(0, os.getcwd())
 sys.path.insert(0, os.path.join(os.getcwd(), os.pardir))
 
-# Import unit test cases or suites here
-from test_tileobj import TileTest
```

Comment:
These were below the `sys.path.insert` calls for a reason; as far as I remember `sys.path.insert` alters the python module path, in this case so modules in the current working directory can be imported too. Might be relevant.

Reply:
Yes I think you're right! That was `autopep8`, I should have made sure that it didn't do that - I'll change it back 👍